ParisBNF/abb/BNFabb51.xml

```diff
@@ -0,0 +1,495 @@
+            <editionStmt>
+                <p/>
+            </editionStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas:  Eine multimediale
+                    Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0303BNF"/>
+                        <collection>Manuscrits orientaux</collection>
+                        <collection>Fonds éthiopien</collection>
+                        <collection>d'Abbadie</collection>
+                        <idno facs="https://gallica.bnf.fr/ark:/12148/btv1b100878602.r=Abbadie%2051?rk=64378;0">BnF Éthiopien d'Abbadie 51</idno>
+                  </msIdentifier>
+
+                    <msContents>
+                      <msItem xml:id="ms_i1">
+                        <locus from="1ra" to="77vb"/>
+                         <title type="complete" ref="LIT2290Sargis"></title>
+                         <textLang mainLang="gez"/>
+                         <incipit xml:lang="gez"><hi rend="rubric">በስመ፡ አብ፡ ወወል</hi>ድ፡ ወመንፈስ፡ ቅዱ<hi rend="rubric">ስ፡
+                           ፩፡ አምላክ፡ ፍቁራንየ፡</hi> ስምዑ፡ ዘ<hi rend="rubric">ኮነ፡ በመዋዕለ፡ ሀ</hi>ርቃል፡ ንጉሠ፡ ሮ<hi rend="rubric">ም፡ በእንተ፡ ብእ</hi>ሲ፡
+                           ዘስሙ፡ ሰርጊ<hi rend="rubric">ስ፡ አብርጋዊ፡ ዘእ</hi>ምሥራዊተ፡ ንጉሥ፡ ናሁ፡ ሤሞ፡ መስፍነ፡ ላዕለ፡ ክልኤ፡ አህጉር፡ አፍራጊያ፡
+                         </incipit>
+                        </msItem>
+
+                         <msItem xml:id="ms_i2">
+                           <locus from="78ra" to="113va"/>
+                            <title type="complete" ref="LIT2461Testam"/>
+                                 <textLang mainLang="gez"/>
+                              <incipit xml:lang="gez"><hi rend="rubric">ኪዳን፡ ዘእግዚእነ፡ ወመ</hi>ድኃኒነ፡
+                              ኢየሱስ፡ ክርስ<hi rend="rubric">ቶስ፡
+                                ወኮነ፡ እንከ፡ እም</hi>ድሕረ፡ ተንሥኣ፡ እግዚ<hi rend="rubric">እነ፡
+                                ኢየሱስ፡ ክርስቶስ፡</hi>እምነ፡ ምዉታን፡ አስተር<hi rend="rubric">ኣየነ፡
+                                ወተገሰ፡
+                                እምቶማ</hi>ስ፡ ወማቴዎስ፡ ወዮሐን<hi rend="rubric">ስ፡ ወተፈዊሰነ፡
+                                ከመ፡ ተ</hi>ንሥኣ፡ ሊቅ፡ ወደቅነ፡
+                              </incipit>
+                             </msItem>
+
+
+                             <msItem xml:id="ms_i3">
+                               <locus from="113va" to="130va"/>
+                                <title type="complete" ref="LIT2462Testam">
```

Comment:
close empty tag or at least delete the space/linebreak, or nothing will be printed
should be 
`<title type="complete" ref="LIT2462Testam"></title>`
or
`<title type="complete" ref="LIT2462Testam"/>`
(or 
`<title type="complete" ref="LIT2462Testam">Testament...</title>`)

Reply:
it is like this, 1st variant, no? :)

Reply:
It is 1st variant, no?

Reply:
no, there is a line break, which has the same meaning as space. no line break

Reply:
please replace by one of the variants proposed with no line break in the middle

Reply:
ok, done